tests/unittests/distros/test_networking.py

```diff
@@ -229,3 +235,113 @@ def test_retrying_and_strict_behaviour(
             5 * len(wait_for_physdevs_netcfg["ethernets"])
             == m_settle.call_count
         )
+
+
+class TestLinuxNetworkingApplyNetworkCfgNames:
```

Comment:
Nice refactor of these tests! This is much nicer.

Reply:
Thanks!